documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2066,19 +2042,13 @@ While using the `pgoutput` plug-in, it is recommended that you configure `filter
 See {link-prefix}:{link-postgresql-plugins}[Logical Decoding Output Plug-in Installation for PostgreSQL] for more detailed instructions for setting up and testing logical decoding plug-ins.
 ====
 
-[NOTE]
-====
-As of {prodname} 0.10, the connector supports PostgreSQL 10+ logical replication streaming by using `pgoutput`.
-This means that a logical decoding output plug-in is no longer necessary and changes can be emitted directly from the replication stream by the connector.
-====
-
 As of PostgreSQL 9.4, the only way to read changes to the write-ahead-log is to install a logical decoding output plug-in. Plug-ins are written in C, compiled, and installed on the machine that runs the PostgreSQL server. Plug-ins use  a number of PostgreSQL specific APIs, as described by the link:https://www.postgresql.org/docs/current/static/logicaldecoding-output-plugin.html[PostgreSQL documentation].
 
-The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:http://www.json.org/[JSON] format.
-See the documentation for your chosen plug-in to learn more about the plug-in's requirements, limitations, and how to compile it.
+The PostgreSQL connector works with one of {prodname}'s supported logical decoding plug-ins to encode the changes in either link:https://github.com/google/protobuf[Protobuf format] or link:https://github.com/postgres/postgres/blob/master/src/backend/replication/pgoutput/pgoutput.c[pgoutput] format.
+`pgoutput` plugin comes out-of-the-box with PostgreSQL server.
+For other plug-in see the documentation for your chosen plug-in to learn more about the plug-in's requirements, limitations, and how to compile it.
```

Comment:
That sounds a bit odd, given that the "other plug-in" is maintained by ourselves.

Reply:
@gunnarmorling Rephrased